widgets/src/dock.rs

```diff
@@ -524,12 +524,12 @@ impl Dock {
             let entry = self.items.get_or_insert(cx, entry_id, | cx | {
                 (template, WidgetRef::new_from_ptr(cx, Some(*ptr)))
             });
-            return Some(entry.1.clone())
+            Some(entry.1.clone())
         }
         else {
-            log!("PortalList template not found {}", template);
+            warning!("Template not found: {template}. Did you add it to the <Dock> instance in `live_design!{{}}`?");
```

Comment:
btw @makepaddev - it would be great to rename this to `warn!` to make it compatible with other rust log implementations. or use one of them and add callback to things like adb, so we can collect logs from external crates too?

a log callback on makepad side would definitely be useful to allow logs to be exported from the user's device.

Reply:
Yeah I'll probably eventually work with the makepad team to implement a `log`-compatible and `tracing`-compatible backend such that other crates in the dependency chain can also output to it.